scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -798,6 +799,34 @@ class CliTest extends AbstractCliTest with CliTestBehavior {
   testCli("1.6.0-RC4") // test for runDynamic
   testCli(stableVersion) // test for runScalafmt
 
+  test(s"path-error") {
```

Comment:
should we add a similar test to the dynamic suite?

Reply:
I think the behaviour will be different in older version, so the test would fail initially.

Reply:
for older versions, we could use a check that is appropriate to them, but for this check start with `nightly` (since it actually dynamically loads the current build).

Reply:
Added two tests to DynamicSuite